rtlib/CMakeLists.txt

```
@@ -23,7 +23,10 @@ set(srcs
     src/ImageTransformResampler.cpp
     src/BSplineLandmarkWarping.cpp
     src/DisegniSegmenter.cpp
-)
+    src/Data.cpp
+    src/Image.cpp
+    src/Envi.cpp
+    src/Mesh.cpp)
```

Nitpick: move the ) to a new line:
```suggestion
    src/Mesh.cpp
)
```